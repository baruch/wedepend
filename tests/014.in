module a;

static if (false) {
    import std.false_cond;
} else {
    import std.true_cond;
}
