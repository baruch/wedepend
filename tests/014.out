std.false_cond
std.true_cond
